src/themes/cornerstone/register_success.js

```diff
@@ -0,0 +1,37 @@
+import { shopgateHideElementByClassName } from '../../modules/ShopgateHideElementByClassName';
+import { shopgateExecuteAppRelatedCode } from '../../modules/ShopgateExecuteAppRelatedCode';
+
+/**
+ * Makes register success page escape proof
+ */
+export function shopgateRegisterSuccess() {
+  shopgateHideElementByClassName('header');
+  shopgateHideElementByClassName('footer');
+  shopgateHideElementByClassName('button');
+
+  /**
+   * Shows the login page in the app
+   */
+  function shopgateAppOpenLoginForm() {
+    const commands = [
```

Comment:
I think this will happen when we finish the implementation of order-placed here or when the trampoline is done. I already started doing something like that when I started to work on the trampoline (see: https://github.com/shopgate/bigcommerce-js/tree/BIGC-627-trampoline/src/modules/app_commands and https://github.com/shopgate/bigcommerce-js/blob/BIGC-627-trampoline/src/modules/ShopgateSendAppCommands.js)

Reply:
I have added a TODO and updated the lib version. Carsten can give maybe more insights